ibm/mas_devops/roles/mongodb/README.md

```diff
@@ -20,6 +20,7 @@ Also, you need to have AWS user credentials configured via `aws configure` comma
 
 To run the `docdb_secret_rotate` MONGODB_ACTION when the provider is `aws` you must have already installed the [Mongo Shell](https://www.mongodb.com/docs/mongodb-shell/install/).
 
+To run this role with provider `community` you must first run the [cluster_monitoring role](https://ibm-mas.github.io/ansible-devops/roles/cluster_monitoring/). The grafana dashboard and prometheus monitoring resources depend on the `cluster_monitoring` role.
```

Comment:
Can we change this statement to say something like "This role will install a GrafanaDashboard used for monitoring the MongoDB instance whe the provider is `community` and you have run the [cluster_monitoring role](https://ibm-mas.github.io/ansible-devops/roles/cluster_monitoring/) previously. If you did not run the [cluster_monitoring role](https://ibm-mas.github.io/ansible-devops/roles/cluster_monitoring/) then the GrafanaDashboard won't be installed."

This is based on the change to guard against installing the grafanadashboard later on.

Reply:
@whitfiea above changes have been made, currently building